query-engine/core/src/executor/mod.rs

```diff
@@ -88,3 +87,27 @@ pub trait TransactionManager {
 pub fn get_current_dispatcher() -> Dispatch {
     tracing::dispatcher::get_default(|current| current.clone())
 }
+
+tokio::task_local! {
+    static REQUEST_NOW: prisma_value::PrismaValue;
+}
+
+/// A timestamp that should be the `NOW()` value for the whole duration of a request. So all
+/// `@default(now())` and `@updatedAt` should use it.
+pub(crate) fn get_request_now() -> prisma_value::PrismaValue {
```

Comment:
add comment: that panics if REQUEST_NOW has not been set with `with_request_now()`.

We could also just return a datetime.

Reply:
Also comment that it would belong in a query context if we had one.